HACKING.md

```diff
@@ -92,6 +92,22 @@ oc annotate secret/resultsdb-auth  \
     jenkins.io/credentials-description="ResultsDB authentication"
 ```
 
+### Create Fedora Matrix notifications secret
+
+Create the maubot authentication secret (available in BitWarden under `CoreOS CI Fedora Matrix Auth`).
+
+```
+MATRIX_BOT_TOKEN=auth_token
+MATRIX_BOT_WEBHOOK_URL=url
+oc create secret generic matrix-bot-webhook-token \
+    --from-literal=username=${MATRIX_BOT_WEBHOOK} \
```

Comment:
Hmm, I think it's a bit weird to model this as a username/password, but not a big deal. I do see the appeal of having it live alongside the token instead of having it live out of band.

That said, this should be `MATRIX_BOT_WEBHOOK_URL` here.

Reply:
... surprise surprise, there, uncomitted : 
```
diff --git a/HACKING.md b/HACKING.md
index b11db21..7373202 100644
--- a/HACKING.md
+++ b/HACKING.md
@@ -98,7 +98,7 @@ oc annotate secret/resultsdb-auth  \
 MATRIX_BOT_TOKEN=auth_token
 MATRIX_BOT_WEBHOOK_URL=url
 oc create secret generic matrix-bot-webhook-token \
-    --from-literal=username=${MATRIX_BOT_WEBHOOK} \
+    --from-literal=username=${MATRIX_BOT_WEBHOOK_URL} \
     --from-literal=password=${MATRIX_BOT_TOKEN}
 oc label secret/matrix-bot-webhook-token \
     jenkins.io/credentials-type=usernamePassword
```

:sweat_smile:

Reply:
@jlebon yeah, i did not find a better matching secret type. Agreed it's weird, having two secrets would be worse IMO

Reply:
https://github.com/coreos/coreos-ci/pull/66